multiqc/modules/rseqc/bam_stat.py

```diff
@@ -76,7 +82,9 @@ def parse_reports(self):
         for s_name in self.bam_stat_data:
             if s_name not in self.general_stats_data:
                 self.general_stats_data[s_name] = dict()
-            self.general_stats_data[s_name].update( self.bam_stat_data[s_name] )
+                #Don't write to general stat if nothing to write..
```

Comment:
This is a little too harsh a test. It's possible to have perfectly valid paired end data which has 0 reads mapped in proper_pairs. Can we not use the same test as above for paired-end data?